MAUI/Scheduler/appointments.md

```diff
@@ -882,6 +882,56 @@ N>
 
 N> [View sample in GitHub](https://github.com/SyncfusionExamples/maui-scheduler-examples/tree/main/RecursiveExceptionAppointment/BusinessObject)
 
+## Suspend and resume the appointment update
+
+Schedule allows you to suspend and resume the appointment UI update while performing collection changes (Add/Remove/Reset).`SuspendAppointmentViewUpdate ` method will suspend appointment UI rendering until you resume it when large number of data added dynamically in schedule `AppointmentSource` to avoid each time updating UI when collection changes. After data added dynamically in schedule, you can call `ResumeAppointmentViewUpdate ` to update the appointment UI rendering. 
```

Comment:
Schedule allows you to suspend and resume the appointment UI update while performing collection changes (Add/Remove/Reset) **to improve UI rendering when working with large appointment collections**

Reply:
updated the changes